### ISSUE TITLE
📡 Problem with the Faucet

### ISSUE THREAD

**Issue by Xatavr** [availproject/polkadot-testnet-faucet#87]
### Faucet Type

Web

### Problem
The wallet balance is 0. failed to fetch
 I won't be able to pass the test.
![Снимок экрана 2024-04-06 201917](https://github.com/availproject/polkadot-testnet-faucet/assets/139537029/c87c0e83-3fff-459b-8e1b-db62fe6faba2)

### Address
5FgYVX3ENeRDUCu8dGQkah6ujzJpAWd8bC9W9cD19ntJ7jPs
_No response_

### Network ID

Relay Chain

**Comment by zeeboo** [comment]
yup same issue. Can you guys send faucet this way, really appriciate the help. 

5FxN9GUXk7k53bpkTNcXYDMtW8VmYC6Y1Z5ENHbxinWFpTBb
